gradle/dependencies.gradle

```diff
@@ -1,11 +1,28 @@
+// Use the .github folder as a way to find the root directory.
+// Without this, the plugins module will not be able to use this script without hacks.
+File rootDirectory = rootDir
+while (!rootDirectory.listFiles().any { it.isDirectory() && it.name == ".github" }) {
+    rootDirectory = rootDirectory.parentFile
+}
+
+apply from: "$rootDirectory/gradle/compose.gradle"
+
+def kotlinVersion
+if (useCompose.toBoolean()) {
```

Comment:
We are free to set `1.4.30` for both regular and compose variants.

Reply:
we certainly can, but perhaps that should be in a different ticket as there might be unexpected changes as a result of this?

At least with the `useCompose` flag, we isolate any potential issues away from the 'stable' binary. Happy to fast follow this PR with an update Kotlin PR. If precede this PR with a Kotlin upgrade.